cds_ils/static_pages/search_guide.html

```diff
@@ -0,0 +1,674 @@
+<h2 id="cds-videos">CERN library catalogue</h2>
```

Comment:
In general, I think the links should redirect to the search page to demonstrate the example

Reply:
Yes, the problem that I encountered is that we don't have a production environment yet. We could make them point to `sandbox` though - and update them later.